test/fw/ptl/utils/pbs_anonutils.py

```diff
@@ -792,23 +803,19 @@ def __add_alias_attr(self, key, alias_key):
         :param alias_key: the alias
         :type alias_key: str
```

Comment:
You're checking to see if key and alias_key are None.  Your :type: says just str.  Should it be str or None?